src/main/java/Game.java

```diff
@@ -114,50 +115,48 @@ private static String giveItem(List<String> items, Player player) {
 		String toWhat = items.get(2);
```

Comment:
This will crash the thread when you "give wallet" for example. It will through a null pointer exception :O.
Be sure to add some try catch blocks or account for this case.